content/en/docs/setup/additional-setup/sidecar-injection/index.md

```diff
@@ -60,14 +60,14 @@ sleep-64c6f57bc8-f5n4x   2/2     Running   0          24s
 
 ### Automatic sidecar injection
 
-Sidecars can be automatically added to applicable Kubernetes pods using a
+When enabled with a namespace label, sidecars can be automatically added to applicable Kubernetes pods using a
```

Comment:
Maybe this change is overkill? It's getting a little repetitive. The following change says the same thing and since this sentence says "Sidecars can ..." (not "Sidecars will ..."), I don't think it's ambiguous here.

Reply:
Fixed